test/models/database/post/post_data_test.dart

```diff
@@ -6,6 +6,7 @@ import "package:proxima/models/database/user/user_id_firestore.dart";
 void main() {
   group("Post Data testing", () {
     test("hash overrides correctly", () {
+      //TODO: moves this mock to package
```

Comment:
I think that we can move this mock now.

I think it also duplicates with `test/models/database/post/post_firestore_test.dart` line 20

+ little typo: `post_ti**l**tle`